### PR TITLE
Set KubeVela as verified publisher of ArtifactHub

### LINKED ISSUE #977: add kubevela helm repo to artifacthub.io
Now, we have our hub https://kubevelacharts.oss-cn-hangzhou.aliyuncs.com/core [registered](https://artifacthub.io/packages/helm/kubevela/vela-core) with repo ID (omitted) into https://artifacthub.io/ .

But it's not enough, we need:

1. add readme to that hub repo.
2. verify to be labeled as Verified Publisher . 
    > If you want your repositories to be labeled as Verified Publisher, you can add a [meta file](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-repo.yml)  to each of them including the repository ID provided below. This label will let users know that you own or have control over the repository. The repository metadata file must be located at the path used in the repository URL.
3. add helm badge into readme `[Artifact HUB](https://artifacthub.io/packages/search?repo=kubevela)`

_This idea was originally posted by @Cweiping in https://github.com/oam-dev/kubevela/issues/942#issuecomment-769519298_

### PR DESCRIPTION
**Pull request by zzxwill**
Set KubeVela team as a verified publisher of Artifacthub,
Add README to the repo and add helm badge in github readme

fix #977

### CODE REVIEW

**Comment by zzxwill** [comment]
Please hold on when README is verified to successfully show in Helm chart page in artifacthub.io.

`Verified Publisher` could work. I will delete `KubeVela PoC` repo later.
image

File: hack/artifacthub/artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: ARTIFACT_HUB_REPOSITORY_ID
+owners:
+  - name: Lei Zhang (Harry)
+    email: resouer@gmail.com
+  - name: Jianbo Sun
+    email: jianbo.sjb@alibaba-inc.com
+  - name: Zheng Xi Zhou
+    email: zzxwill@gmail.com
```
Comment (ryanzhang-oss) [inline hack/artifacthub/artifacthub-repo.yml:19]:
```suggestion
    email: zzxwill@gmail.com
  - name: Ryan Zhang
    email: yangzhangrice@hotmail.com 
```

File: hack/artifacthub/artifacthub-repo.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: ARTIFACT_HUB_REPOSITORY_ID
+owners:
+  - name: Lei Zhang (Harry)
+    email: resouer@gmail.com
+  - name: Jianbo Sun
+    email: jianbo.sjb@alibaba-inc.com
```
Comment (wonderflow) [inline hack/artifacthub/artifacthub-repo.yml:17]:
```suggestion
    email: wonderflow.sun@gmail.com
```

File: hack/artifacthub/artifacthub-repo.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - name: Zheng Xi Zhou
+    email: zzxwill@gmail.com
+  - name: Ryan Zhang
+    email: yangzhangrice@hotmail.com
```
Comment (wonderflow) [inline hack/artifacthub/artifacthub-repo.yml:21]: please also add @hongchaodeng here

**Review by wonderflow — APPROVED** [review]
Good Job, thanks. Please feel free to merge when it's ready.

**Comment by zzxwill** [comment]
I have verified README could show as expected.
image

**Pull request merged**